### PR TITLE
Provide variable for optional alert creation

### PR DESCRIPTION
**Pull request by doncem**
~Straight-forward use case example: only enable alerting for production.~ Do not create an alert if module provides `enabled=false`

Keeping default as true for backwards compatibility

### CODE REVIEW

File: main.tf
Comment (timja) [inline main.tf]: What do you think about not creating the alert if it's disabled?
i.e. do a count = 0 on the template resource

Comment (doncem) [reply main.tf]: forgot about that feature. let me update the PR then

File: variables.tf
Comment (timja) [inline variables.tf]:
```suggestion
  description = "Whether alert is created or not"
```

**Review by timja — APPROVED** [review]
LGTM assuming you've tested it...
If not please test before merging

**Comment by doncem** [comment]
Proved it's working with bulk scan shared infra - PR linked in this one

**Pull request merged**
